Wikipedia/Code/WMFWelcomeAnalyticsViewController.swift

```diff
@@ -17,29 +36,40 @@ class WMFWelcomeAnalyticsViewController: UIViewController {
             toggle.isOn = false
             BITHockeyManager.shared().crashManager.crashManagerStatus = .alwaysAsk
         }
-        self.view.wmf_configureSubviewsForDynamicType()
+        view.wmf_configureSubviewsForDynamicType()
+        
+        linkLabel.addGestureRecognizer(UITapGestureRecognizer.init(target: self, action: #selector(showPrivacyAlert(_:))))
```

Comment:
If possible, this should be a button instead of a label with a tap gesture recognizer.

Reply:
Formerly was having trouble getting buttons with longer strings to wrap lines properly (w long translations and or big dynamic type size), but I can re-check to see if this is still an issue.